src/bundle/Resources/public/scss/_buttons.scss

```
@@ -171,9 +171,11 @@
         &:hover {
             color: $ibexa-color-white;
             box-shadow: 0 0 0 transparent, $ibexa-btn-hover-box-shadow-info;
+            background-color: $ibexa-color-info;
 
             &:focus {
                 box-shadow: $ibexa-btn-focus-box-shadow-info, $ibexa-btn-hover-box-shadow-info;
+                color: $ibexa-color-white;
```

```suggestion
                color: $ibexa-color-white;
                box-shadow: $ibexa-btn-focus-box-shadow-info, $ibexa-btn-hover-box-shadow-info;
```
for consistency of attributes order? ;)